edb/schema/functions.py

```diff
@@ -618,10 +620,30 @@ def get_ast(self, schema: s_schema.Schema) -> List[qlast.FuncParam]:
 
 class VolatilitySubject(so.Object):
 
+    # `volatility` is the specified volatility of an object, and is
+    # named `volatility` instead of `specified_volatility` because we
+    # write `SET volatility := ...`;
+    #
+    # We keep `inferred_volatility` separate from the specified
+    # volatility (instead of populating an unspecified volatility
+    # field directly with the inferred volatility) because we don't
+    # want the `DESCRIBE` output to include `SET volatility := ...`
+    # when the volatility was inferred.
     volatility = so.SchemaField(
-        ft.Volatility, default=ft.Volatility.Volatile,
+        ft.Volatility, default=None,
         compcoef=0.4, coerce=True, allow_ddl_set=True)
 
+    inferred_volatility = so.SchemaField(
+        ft.Volatility, default=None)
+
+    def get_effective_volatility(
```

Comment:
This isn't properly documented, but you are actually allowed to overload the normal getter for a field value, and so this should be a `def get_volatility()` instead.  See `Constraint.get_subject()` for an example on how to get to the raw field value in this case.  Oh, and the fallback to `Volatile` is likely not needed anymore: the volatility should always either be inferred, or specified explicitly (in the case of SQL-backed functions in std).